docker/config/configuration.json

```diff
@@ -0,0 +1,166 @@
+      "type" : "string",
+      "value" : "North Rhine-Westphalia"
+    },
+    "serviceProvider.positionName" : {
+      "type" : "string",
+      "value" : "TBA"
+    },
+    "serviceProvider.hoursOfService" : {
+      "type" : "string",
+      "value" : "9 to 5"
+    },
+    "serviceProvider.contactInstructions" : {
+      "type" : "string",
+      "value" : "Call or email, please!"
+    },
+    "serviceProvider.onlineResource" : {
+      "type" : "string",
+      "value" : "team website|https://github.com/orgs/52North/teams/geoprocessing/members"
+    },
+    "serviceProvider.role.value" : {
+      "type" : "string",
+      "value" : "resourceProvider"
+    },
+    "serviceProvider.role.codespace" : {
+      "type" : "uri",
+      "value" : "http://www.isotc211.org/2005/resources/Codelist/gmxCodelists.xml#CI_RoleCode"
+    },
+    "serviceIdentification.file" : {
+      "type" : "file",
+      "value" : null
+    },
+    "serviceIdentification.abstract" : {
+      "type" : "multilingual",
+      "value" : {
+        "en" : "52North WPS 2.0 Service",
+        "de" : "52North WPS 2.0 Dienst"
+      }
+    },
+    "serviceIdentification.title" : {
+      "type" : "multilingual",
+      "value" : {
+        "en" : "javaPS",
+        "de" : "javaPS"
+      }
+    },
+    "serviceIdentification.keywords" : {
+      "type" : "string",
+      "value" : "OGC, WPS 2.0"
+    },
+    "serviceIdentification.serviceType" : {
+      "type" : "string",
+      "value" : "OGC:TEST"
+    },
+    "i18n.defaultLanguage" : {
+      "type" : "string",
+      "value" : "en"
+    },
+    "serviceIdentification.serviceTypeCodeSpace" : {
+      "type" : "string",
+      "value" : null
+    },
+    "serviceIdentification.fees" : {
+      "type" : "string",
+      "value" : "NONE"
+    },
+    "serviceIdentification.accessConstraints" : {
+      "type" : "string",
+      "value" : "NONE"
+    },
+    "misc.includeStackTraceInExceptionReport" : {
+      "type" : "boolean",
+      "value" : true
+    },
+    "misc.characterEncoding" : {
+      "type" : "string",
+      "value" : "UTF-8"
+    },
+    "i18n.showAllLanguageValues" : {
+      "type" : "boolean",
+      "value" : false
+    },
+    "statistics.counting-outputstream" : {
+      "type" : "boolean",
+      "value" : false
+    },
+    "service.serviceURL" : {
+      "type" : "uri",
+      "value" : "http://localhost:8080/wacodis-wps/service"
+    },
+    "filewatcher.enabled" : {
+      "type" : "boolean",
+      "value" : true
+    },
+    "misc.includeOriginalRequest" : {
+      "type" : "boolean",
+      "value" : false
+    },
+    "misc.baseDirectory" : {
+      "type" : "file",
+      "value" : "/tmp/wps/outputs/"
+    },
+    "misc.duration" : {
+      "type" : "string",
+      "value" : "PT2H"
+    },
+    "misc.checkInterval" : {
+      "type" : "string",
+      "value" : "PT1H"
+    },
+    "service.request.timeout" : {
+      "type" : "integer",
+      "value" : 0
+    }
+  }
+}
```

Comment:
Due to the new possbility of configuring the javaps by the use of envs, this configuration file, which will be provided to the docker container by file volume binding, would be unnecessary, or?

Reply:
most likely, yes.